lib/tasks/dataverse_migration.rake

```diff
@@ -55,26 +55,26 @@ namespace :migration do
       MigrationLogger.info "Processing the object #{object_id}"
       #reading the metadata file
       metadata_file = Nokogiri::XML(File.open(file))
-      metadata = metadata_file.xpath("//ddi_to_dcterms", MigrationConstants::NS)
+      metadata = metadata_file.xpath("//ddi_to_dcterms", CommonConstants::NS)
```

Comment:
CommonConstants is here:

https://github.com/ualbertalib/HydraNorth/blob/master/lib/tasks/constants/common_constants.rb

It looks like MigrationConstants and some other constants were merged into it as part of this commit:

https://github.com/ualbertalib/HydraNorth/commit/119acc300fae5514824bffd6d4b6bd301a607d43#diff-178d2afef07858d1f2e00cb3662df5bcR1